[2024-10-12T18:16:27.102+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-12T18:16:27.114+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: first-dag.task2 scheduled__2022-12-25T00:00:00+00:00 [queued]>
[2024-10-12T18:16:27.122+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: first-dag.task2 scheduled__2022-12-25T00:00:00+00:00 [queued]>
[2024-10-12T18:16:27.122+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-10-12T18:16:27.158+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): task2> on 2022-12-25 00:00:00+00:00
[2024-10-12T18:16:27.163+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=4091) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-12T18:16:27.164+0000] {standard_task_runner.py:72} INFO - Started process 4098 to run task
[2024-10-12T18:16:27.164+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'first-dag', 'task2', 'scheduled__2022-12-25T00:00:00+00:00', '--job-id', '932', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpjnyz6hbt']
[2024-10-12T18:16:27.165+0000] {standard_task_runner.py:105} INFO - Job 932: Subtask task2
[2024-10-12T18:16:27.204+0000] {task_command.py:467} INFO - Running <TaskInstance: first-dag.task2 scheduled__2022-12-25T00:00:00+00:00 [running]> on host 50fe842129eb
[2024-10-12T18:16:27.278+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='james' AIRFLOW_CTX_DAG_ID='first-dag' AIRFLOW_CTX_TASK_ID='task2' AIRFLOW_CTX_EXECUTION_DATE='2022-12-25T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-12-25T00:00:00+00:00'
[2024-10-12T18:16:27.279+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-12T18:16:27.295+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-10-12T18:16:27.296+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Running task2"']
[2024-10-12T18:16:27.304+0000] {subprocess.py:86} INFO - Output:
[2024-10-12T18:16:27.305+0000] {subprocess.py:93} INFO - Running task2
[2024-10-12T18:16:27.306+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-12T18:16:27.325+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-12T18:16:27.326+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=first-dag, task_id=task2, run_id=scheduled__2022-12-25T00:00:00+00:00, execution_date=20221225T000000, start_date=20241012T181627, end_date=20241012T181627
[2024-10-12T18:16:27.386+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-12T18:16:27.403+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-12T18:16:27.404+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
